### DISCUSSION TITLE
How to auto restart server when UserSeeder.js is changed?

### DISCUSSION THREAD

**Discussion by HK770** [adonisjs/core#1452] [Help]
Hello. I am using adonis serve --dev --polling to start the adonis server on my mac terminal. However, when I change the UserSeeder.js (which I use to add data to my database), I have to hit "Command" + "." on my keyboard (i.e., stop the adonis server), run adonis seed, and then restart the server. Is there a way so that I don't have to restart the server this way?

**Comment by McSneaky** [comment]
Easiest is to open up other terminal and run commands in there
Most likely could run ace commands in some hook too (like "boot"), but it's cleaner just to open new terminal :)
